### DISCUSSION TITLE
Not able to pull the docker image of apache/superset..Not even the latest as well as the 2.0.1 versionSte

### DISCUSSION THREAD

**Discussion by falgunifb** [apache/superset#24470] [Q&A / Help]
Steps to reproduce the error: 
 1. docker pull apache/superset:2.1.0
 Error :
![MicrosoftTeams-image](https://github.com/apache/superset/assets/52688344/3bb403bd-0125-451c-840b-bffd4cd44421)

**Comment by sfirke** [comment]
That command looks valid.  Looks like you're being denied by Docker Hub.  Does this help? https://stackoverflow.com/a/32845133

**Reply by falgunifb** [reply]
Getting ERROR: failed to solve: apache/superset:2.1.0: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/apache/superset/blobs/sha256:27d3d51cc65ebcbea00866e03f5e9af1b7c873d55139d69cfa3330d243729595: 403 Forbidden

**Reply by sfirke** [reply]
`docker pull apache/superset:latest` succeeds for me.  Your access to docker hub is being blocked.  Maybe because of the country you're accessing it from, as suggested in the link I shared, or maybe by something else.  Try `docker pull hello-world` and I expect you'll encounter the same thing, which would confirm this is not a Superset issue.

**Reply by falgunifb** [reply]
maybe some firewall issue working fine on ec2 thanks
